api-server/src/server/boot/authentication.js

```diff
@@ -188,3 +194,40 @@ function createGetPasswordlessAuth(app) {
     );
   };
 }
+
+function mobileLogin(app) {
+  const {
+    models: { User }
+  } = app;
+  return async function getPasswordlessAuth(req, res, next) {
+    const {
+      headers: { token: accessToken }
+    } = req;
+
+    const auth0Res = await fetch(
+      `https://${process.env.AUTH0_DOMAIN}/userinfo`,
+      {
+        headers: { Authorization: `Bearer ${accessToken}` }
+      }
+    );
+
+    const { email } = await auth0Res.json();
+
+    if (!isEmail(email)) {
+      return next(
+        wrapHandledError(new TypeError('decoded email is invalid'), {
+          type: 'info',
+          message: 'The email encoded in the link is incorrectly formatted',
+          redirectTo: `${origin}/signin`
+        })
+      );
+    }
+
+    return User.findOne$({ where: { email: email } })
+      .do(async user => {
+        const token = await user.mobileLoginByRequest(req, res);
+        return res.json(token);
```

Comment:
```suggestion
        await user.mobileLoginByRequest(req, res);
        res.end();
```
Since the token is getting added as a cookie I don't believe there's any need to send it back in the body.